src/main/java/cubicchunks/world/ICubeCache.java

```diff
@@ -23,14 +23,19 @@
  */
 package cubicchunks.world;
 
+import cubicchunks.util.CubeCoords;
 import cubicchunks.world.column.Column;
 import cubicchunks.world.cube.Cube;
 
 public interface ICubeCache {
 	boolean cubeExists(int cubeX, int cubeY, int cubeZ);
 
+	boolean cubeExists(CubeCoords coords);
```

Comment:
this can be default method with `return this.getCube(coords) == null`. Client should contain all cubes, and return BlankCube for cubes that don't exist.